mr/src/test/java/org/apache/iceberg/mr/hive/HiveIcebergTestUtils.java

```diff
@@ -218,13 +218,12 @@ public static void assertEquals(Record expected, Record actual) {
     for (int i = 0; i < expected.size(); ++i) {
       if (expected.get(i) instanceof OffsetDateTime) {
         // For OffsetDateTime we just compare the actual instant
-        Assert.assertEquals(
-            ((OffsetDateTime) expected.get(i)).toInstant(),
-            ((OffsetDateTime) actual.get(i)).toInstant());
+        Assertions.assertThat(((OffsetDateTime) actual.get(i)).toInstant())
+            .isEqualTo(((OffsetDateTime) expected.get(i)).toInstant());
       } else if (expected.get(i) instanceof byte[]) {
-        Assert.assertArrayEquals((byte[]) expected.get(i), (byte[]) actual.get(i));
+        Assertions.assertThat((byte[]) actual.get(i)).isEqualTo((byte[]) expected.get(i));
```

Comment:
we can remove this line and `else if (expected.get(i) instanceof byte[]) {` because AssertJ can properly handle byte array comparisons

Reply:
done